src/native/aws_signing.c

```diff
@@ -249,6 +252,14 @@ static void s_aws_chunk_signing_complete(struct aws_signing_result *result, int
     s_cleanup_callback_data(callback_data);
 }
 
+static void s_aws_chunk_signing_complete(struct aws_signing_result *result, int error_code, void *userdata) {
+    s_aws_chunk_like_signing_complete(result, error_code, userdata);
+}
+
+static void s_aws_trailing_headers_signing_complete(struct aws_signing_result *result, int error_code, void *userdata) {
+    s_aws_chunk_like_signing_complete(result, error_code, userdata);
+}
```

Comment:
Trivial: maybe instead of this two functions, just pass the `s_aws_chunk_like_signing_complete` to both chunk and trailing headers signing.

Reply:
Bret suggested this, so that it would be clear I was not accidentally passing the same function to both trailing headers callback, and chunk call back.